test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/FileStoreConnectorSpec.scala

```diff
@@ -164,16 +167,36 @@ class FileStoreConnectorTest extends UnitSpec with WiremockTestServer with Mocki
       )
 
       await(connector.get(Seq(att1, att2))) shouldBe Seq(
-        FileMetadata(
+        FilestoreResponse(
           id = "id",
           fileName = "name",
           mimeType = "text/plain",
           url = Some("url"),
-          scanStatus = Some(ScanStatus.READY),
-          lastUpdated = LocalDate.of(2019, 1, 1).atStartOfDay().toInstant(ZoneOffset.UTC)
+          scanStatus = Some(ScanStatus.READY)
         )
       )
     }
   }
 
+  "Upload" in {
+    stubFor(
+      post("/file")
+        .willReturn(
+          aResponse()
+            .withStatus(Status.ACCEPTED)
+            .withBody(fromResource("filestore/binding-tariff-filestore_upload-response.json"))
+        )
+    )
+
+    val file = FileUpload(TemporaryFile("example-file.txt"), "file.txt", "text/plain")
+
+    val result = await(connector.upload(file))
+
+    result shouldBe FilestoreResponse(
+      id = "id",
+      fileName = "file-name.txt",
+      mimeType = "text/plain"
+    )
```

Comment:
You can do `verifyPostRequestedFor` with Wiremock to verify that the right data and form parts are sent